### ISSUE TITLE
Fix the "Customizing Colors" documentation section

### ISSUE THREAD

**Issue by nicoddemus** [pytest-dev/pytest#11596]
### Discussed in https://github.com/pytest-dev/pytest/discussions/11573

The section https://docs.pytest.org/en/7.4.x/how-to/logging.html#customizing-colors has a broken example. The "broken" means that it can no be copy/pasted to try and there are no explanations about how to use it.
```py
@pytest.hookimpl
def pytest_configure(config):
    logging_plugin = config.pluginmanager.get_plugin("logging-plugin")

    # Change color on existing log level
    logging_plugin.log_cli_handler.formatter.add_color_level(logging.INFO, "cyan")

    # Add color to a custom log level (a custom log level `SPAM` is already set up)
    logging_plugin.log_cli_handler.formatter.add_color_level(logging.SPAM, "blue")
```
```
INTERNALERROR> Traceback (most recent call last):
INTERNALERROR>   File "/opt/pysetup/.venv/lib/python3.11/site-packages/_pytest/main.py", line 267, in wrap_session
INTERNALERROR>     config._do_configure()
INTERNALERROR>   File "/opt/pysetup/.venv/lib/python3.11/site-packages/_pytest/config/__init__.py", line 1053, in _do_configure
INTERNALERROR>     self.hook.pytest_configure.call_historic(kwargs=dict(config=self))
INTERNALERROR>   File "/opt/pysetup/.venv/lib/python3.11/site-packages/pluggy/_hooks.py", line 514, in call_historic
INTERNALERROR>     res = self._hookexec(self.name, self._hookimpls, kwargs, False)
INTERNALERROR>           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
INTERNALERROR>   File "/opt/pysetup/.venv/lib/python3.11/site-packages/pluggy/_manager.py", line 115, in _hookexec
INTERNALERROR>     return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
INTERNALERROR>            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
INTERNALERROR>   File "/opt/pysetup/.venv/lib/python3.11/site-packages/pluggy/_callers.py", line 113, in _multicall
INTERNALERROR>     raise exception.with_traceback(exception.__traceback__)
INTERNALERROR>   File "/opt/pysetup/.venv/lib/python3.11/site-packages/pluggy/_callers.py", line 77, in _multicall
INTERNALERROR>     res = hook_impl.function(*args)
INTERNALERROR>           ^^^^^^^^^^^^^^^^^^^^^^^^^
INTERNALERROR>   File "/opt/pysetup/tests/conftest.py", line 49, in pytest_configure
INTERNALERROR>     logging_plugin.log_cli_handler.formatter.add_color_level(logging.INFO, "cyan")
INTERNALERROR>     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
INTERNALERROR> AttributeError: 'NoneType' object has no attribute 'log_cli_handler'
```

## Expected

Add a small working code example or clarify instructions.

**Comment by nicoddemus** [comment]
I think the example can be fixed simply by using `config.pluginmanager.get_plugin("logging")`.

**Comment by nicoddemus** [comment]
@aarushis18 sure!

Please also make sure the updated example works.

Also in the future feel free to go ahead and open a PR for unassigned issues like this. 👍

**Comment by touilleWoman** [comment]
Hello, this issue can be closed. Problem solved with PR #11652.
Sorry I mentioned the wrong issue in my PR, I wrote " close #11573" instead of "close #11596".
